hooks/collector.py

```diff
@@ -191,12 +197,22 @@ def process_file(self, settings, parent_item, path):
             for the supplied path
         """
 
+        publish_templates_setting = settings.get("Publish Templates")
+        if publish_templates_setting:
+            publish_templates = publish_templates_setting.value
+        else:
+            publish_templates = {}
```

Comment:
Maybe the variable could be just initialized

```suggestion
        publish_templates = {}
        if publish_templates_setting:
            publish_templates = publish_templates_setting.value
            
```